docs/components/dartpad/progress_indicators/theme/main.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(App());
+}
+
+class App extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Theme Progress Indicator Demo',
+      debugShowCheckedModeBanner: false,
+      theme: _buildShrineTheme(),
```

Comment:
The included screenshot colors seem off. You might need to pass in a shrine color to the linear indicator so it doesn't have the blue part.

Reply:
Done